src/components/CommonButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames'
+import styles from './index.module.scss'
+
+type Props = {
+  name: string
+  onClick: () => void
+  isDisabled?: boolean
+  className?: string
+  // isLoading?: boolean
+}
+
+export default (prop: Props) => {
+  const { name, onClick, isDisabled, className } = prop
+  return (
+    <button
+      type="button"
+      className={classNames(styles.container, isDisabled && styles.isDisabled, className)}
```

Comment:
`disabled` is a valid attribute of button tag

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled